File: typescript/fizzbuzz.ts
const range = (s: number, e: number) => Array.from({ length: (e - s) }, (v, k) => k + s);

function fizzbuzz() {
    for (let i of range(1, 100)) {
        if (i % 15 == 0) {
            console.log("FizzBuzz");
        } else if (i % 3 == 0) {
            console.log("Fizz");
        } else if (i % 5 == 0) {
            console.log("Buzz");
        } else {
            console.log(i);
        }
    }
}

fizzbuzz();

File: c++/fizzbuzz.cxx
#include <iostream>

using std::cout;
using std::endl;

int main(int argc, char** argv)
{
    for (int i = 1; i <= 100; i++)
    {
        if (i % 15 == 0) 
        {
            cout << "FizzBuzz" << endl;
        }
        else if (i % 3 == 0)
        {
            cout << "Fizz" << endl;
        }
        else if (i % 5 == 0)
        {
            cout << "Buzz" << endl;
        }
        else 
        {
            cout << i << endl;
        }
    }
}